src/modules/form/components/group/TableGroup.tsx

```diff
@@ -0,0 +1,150 @@
+import {
+  Box,
+  Stack,
+  Table,
+  TableCell,
+  TableHead,
+  TableRow,
+  Typography,
+} from '@mui/material';
+
+import { useMemo } from 'react';
+import { GroupItemComponentProps } from '../../types';
+import RequiredLabel from '../RequiredLabel';
+import { ItemType } from '@/types/gqlTypes';
+
+/**
+ * Dynamically renders children in a table. See expected format below.
+ *
+ * The TableHeader labels and helper text will be pulled from the labels defined
+ * on the items in the first "row group".
+ *
+ * {
+ *   "type": "GROUP",
+ *   "component": "TABLE",
+ *   "item": [
+ *      {
+ *        "type": "GROUP", // represents first row
+ *        "item": [
+ *            // row 1 col 1 input
+ *            // row 1 col 2 input
+ *            // row 1 col 3 input
+ *        ]
+ *      },
+ *      {
+ *        "type": "GROUP", // represents second row
+ *        "item": [
+ *            // row 2 col 1 input
+ *            // row 2 col 2 input
+ *            // row 2 col 3 input
+ *        ]
+ *      }
+ *    ]
+ * }
+ */
+const TableGroup = ({
```

Comment:
Cool! What will this generic table be used for? Do we ultimately want it to replace some of the more specific tables, like the Disability Table?

Reply:
I think the DisbilityTable needs to stick around, at least for now, because it contains custom logic specific to the HUD disability fields.

This will just be for any inputs that should be rendered as a table. We need it for one of the [custom assessments](https://www.pivotaltracker.com/story/show/187005469) were building out.